index.html

```diff
@@ -204,19 +209,48 @@ <h4>Languages</h4> <img class="arrow" src="images/Disabled.png" alt="">
                         <li class="language"><img src="images/Ellipse_css.png" alt="">CSS</li>
                     </ul>
                 </li>
-                <li><img class="separator"img src="images/Separator Bottom.png" alt=""></li>
+                <li><img class="separator" img src="images/Separator Bottom.png" alt=""></li>
                 <li>
-                    <div class="frame"><h4>Frameworks</h4> <img class="arrow" src="images/right.png" alt="">
+                    <div class="frame">
+                        <h4>Frameworks</h4> <img class="arrow" src="images/right.png" alt="">
                     </div>
                 </li>
-                <li><img class="separator"img src="images/Separator Bottom.png" alt=""></li>
+                <li><img class="separator" img src="images/Separator Bottom.png" alt=""></li>
                 <li>
-                    <div class="frame"><h4>Skills</h4><img class="arrow" src="images/right.png" alt=""></div>
+                    <div class="frame">
+                        <h4>Skills</h4><img class="arrow" src="images/right.png" alt="">
+                    </div>
                 </li>
             </ul>
         </div>
     </section>
     <!-- end of about section -->
-</body>
 
+    <!-- contact form -->
+    <section class="contact_form">
+        <div class="contact_wrapper">
+            <h1>Contact me</h1>
```

Comment:
- [OPTIONAL] I recommend you use the `h2` tag for secondary titles on your page since the `h1` should be used only for the main title which is on the first section of your page.